dkp-info.md

```diff
@@ -2,131 +2,141 @@
+- **5 DKP** will be awarded for being present in the raid before the first pull.
+- **60 DKP** will be given as a seed to new raid members at the end of their first raid in our DKP system.
 
 ### World Bosses
 
-The Industrial Peons are an involved member of the Grobbulus Horde Coalition. This Coalition was set up to help solve Honor farming issues, inform guilds of events/world buff timers, and provide a strong platform to coordinate multiple guilds during world pvp events. The coalition strives to have multiple raids present at each world boss spawn, and has been successful at securing and sharing its lucritive loot. When a World Boss spawns, we encourage members to support the coalition defeat the different bosses and Alliance alike. If a member is present in a world boss raid group and has connected to the Grobbulus Guild Coordination discord server during the fight, 3DKP will be awarded to that member for their assistance. Guild officers will be able to help any new member get set up on the Coalition discord server.
+While the Peons are not a part of any official world boss coalition, we do try to actively join in with them for world boss kills. Each world boss attempt is worth **3 DKP**, regardless of if we kill the boss.
 
-## DKP BASICS
+### Peonbank Weekly Quest
 
-**We use MonolithDKP**. All members who ever plan to spend the DKP they earn with us should have it, and keep it up to date.
+To help improve the resources available to the guild through the guild bank, additional DKP is available to members who complete the weekly guild quest. This guild quest can be completed in either a gold donation or by joining a guild farming session (such as a trash farm for valuable BOEs to sell), and comes in three tiers:
 
-It can be downloaded at [https://www.curseforge.com/wow/addons/monolith-dkp](https://www.curseforge.com/wow/addons/monolith-dkp) or with the Twitch Client. Deadly Boss Mods, ClassicThreatMeter, and Details! are also required addons for raiding.
+- **Tier 1: 8 DKP.** Hitting tier 1 requires a **25g** donation, or **1 hour** of guild farming activity.
+- **Tier 2: 12 DKP.** Hitting tier 2 requires a **50g** donation, or **2 hours** of guild farming activity.
+- **Tier 3: 15 DKP.** Hitting tier 3 requires a **100g** donation, or **3 hours** of guild farming activity.
 
-To check your DKP, simply install the addon and type /dkp. You may also type !dkp into a raid and every officer in the raid will PM you your dkp.
+Only the highest eligible tier will apply, and the Peonbank weekly quest resets every Saturday at noon.
+
+## How Dkp Is Spent
+
+**We use MonolithDKP**. All members who ever plan to spend the DKP they earn with us should have it, and keep it up to date. It can be downloaded at [https://www.curseforge.com/wow/addons/monolith-dkp](https://www.curseforge.com/wow/addons/monolith-dkp) or with any number of WoW addon managers, such as WoWUp.

```

Comment:
With the addon installed, full DKP data can be viewed in game. Raiders who are not in Industrial Peons, however, will not be able to receive sync data due to addon permissions.